libs/spark/src/Unstable_SectionMessage/Unstable_SectionMessage.tsx

```diff
@@ -0,0 +1,211 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import Paper, { PaperProps } from '@material-ui/core/Paper';
+import {
+  Unstable_AlertOctagon,
+  Unstable_AlertTriangle,
+  Unstable_CheckCircle2,
+  Unstable_Cross,
+  Unstable_Info,
+} from '../internal';
+import makeStyles from '../makeStyles';
+import { StandardProps } from '../utils';
+
+interface Unstable_SectionMessageTitleProps
+  extends StandardProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    Unstable_SectionMessageTitleClassKey
+  > {
+  /**
+   * The content of the component.
+   */
+  children?: string;
+}
+
+type Unstable_SectionMessageTitleClassKey = 'root';
+
+const useStylesTitle = makeStyles<Unstable_SectionMessageTitleClassKey>(
+  (theme) => ({
+    root: {
+      ...theme.unstable_typography.label,
+      color: theme.unstable_palette.text.heading,
+      marginBottom: 4,
+    },
+  }),
+  { name: 'MuiSparkUnstable_SectionMessageTitle' }
+);
+
+const Unstable_SectionMessageTitle = function Unstable_SectionMessageTitle(
+  props: Unstable_SectionMessageTitleProps
+) {
+  const { children, className } = props;
+
+  const classes = useStylesTitle();
+
+  return <div className={clsx(classes.root, className)}>{children}</div>;
+};
+
+export type Color = 'success' | 'info' | 'warning' | 'error';
+
+export interface Unstable_SectionMessageProps
+  extends StandardProps<
+    PaperProps,
+    Unstable_SectionMessageClassKey,
+    'variant'
+  > {
+  /**
+   * The action to display. It renders after the message, at the end of the section message.
+   */
+  action?: React.ReactNode;
+  /**
+   * Override the default label for the *close popup* icon button.
+   *
+   * For localization purposes, you can use the provided [translations](/guides/localization/).
+   */
+  closeText?: string;
+  /**
+   * The severity of the section message. This defines the color and icon used.
+   */
+  severity?: Color;
+  /**
+   * Override the icon displayed before the children.
+   * Unless provided, the icon is mapped to the value of the `severity` prop.
+   */
+  icon?: React.ReactNode | false;
+  /**
+   * The ARIA role attribute of the element.
+   */
+  role?: string;
+  /**
+   * Callback fired when the component requests to be closed.
+   * When provided and no `action` prop is set, a close icon button is displayed that triggers the callback when clicked.
+   *
+   * @param {object} event The event source of the callback.
+   */
+  onClose?: (event: React.SyntheticEvent) => void;
+  /**
+   * Optional title to display in the section message.
+   */
+  title?: string;
+}
+
+export type Unstable_SectionMessageClassKey =
+  | 'root'
+  | 'icon'
+  | 'message'
+  | 'action';
```

Comment:
```suggestion
export type Unstable_SectionMessageClassKey =
  | 'root'
  | 'icon'
  | 'message'
  | 'action'
  | 'title'
```
(Part of combining into one component)